runtime/near-vm-runner/src/cache.rs

```diff
@@ -73,11 +73,11 @@ fn cache_error(
 #[cfg(target_arch = "x86_64")]
 pub fn into_vm_result<T>(
     res: Result<Result<T, CompilationError>, CacheError>,
-) -> Result<T, VMError> {
+) -> Result<T, Result<FunctionCallError, VMRunnerError>> {
```

Comment:
Hm, this feels like the nesting here is wrong. I'd expect it to read `Result<Result<T, FunctionCallError> , VMRunnerError>`, so that the call-site can ? the VM runner error?

Reply:
right, your nesting makes much more sense, changed it now